app/components/Pagination.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+
+import { cx } from 'class-variance-authority';
+
+import { Button, Icon } from '@swapr/ui';
+
+interface PaginationProps {
+  isFinalPage: boolean;
+  page: number;
+  setPage: (page: number) => void;
+}
+
+export const Pagination = ({ isFinalPage, page, setPage }: PaginationProps) => (
+  <div className="flex w-full items-center justify-between bg-surface-surface-1 p-4">
```

Comment:
let's remove the bg color 

```suggestion
  <div className="flex w-full items-center justify-between p-4">
```